src/wizards/configureschedule/schedule.js

```
@@ -93,11 +94,6 @@ export class Schedule extends Step {
         return {valid: valid, reasons: reasons, fields: fields};
     }
 
-    @computedFrom('data.repeat')
-    get repeatReset() {
-        this.scheduleError = false; // Easy workaround
```

Did you validate that when an invalid repeat was set, the schedule error was displayed and that as soon as you made a change to the repeat the error was hidden?

---

Yes, the properties are properly bidden now